src/main/java/com/metamx/common/io/smoosh/SmooshedFileMapper.java

```diff
@@ -131,10 +132,20 @@ public ByteBuffer mapFile(String name) throws IOException
   }
 
   @Override
-  public void close() throws IOException
+  public void close()
   {
+    Throwable thrown = null;
     for (MappedByteBuffer mappedByteBuffer : buffersList) {
-      ByteBufferUtils.unmap(mappedByteBuffer);
+      try {
+        ByteBufferUtils.unmap(mappedByteBuffer);
+      } catch (Throwable t) {
```

Comment:
In this particular case I think the behavior is OK due to the very limited nature of what is accomplished, or intended, by the unmap.